<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>communicationTypeWidget</class>
 <widget class="QWidget" name="communicationTypeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>60</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="UDP_RS485_Widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>60</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>60</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>999</width>
     <height>71</height>
    </size>
   </property>
   <widget class="QStackedWidget" name="UDP_RS485_stacked">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>5</y>
      <width>281</width>
      <height>61</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="UDP">
     <widget class="QLabel" name="label_80">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>47</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Порт</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_81">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>0</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Таймаут,мс</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="udpPort">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>111</width>
        <height>30</height>
       </rect>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <property name="decimals">
       <number>0</number>
      </property>
      <property name="maximum">
       <double>9999999999999999635896294965248.000000000000000</double>
      </property>
      <property name="value">
       <double>4000.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="udpTimeout">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>161</width>
        <height>30</height>
       </rect>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <property name="decimals">
       <number>0</number>
      </property>
      <property name="maximum">
       <double>300.000000000000000</double>
      </property>
      <property name="value">
       <double>300.000000000000000</double>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="RS485">
     <widget class="QComboBox" name="Num3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>281</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>66</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>999</width>
        <height>30</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_82">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>COM-Порт</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QComboBox" name="udpUse">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>25</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string> RS485</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string> UDP</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_92">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>4</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Опрос</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_83">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>4</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>IP-Адрес</string>
    </property>
   </widget>
   <widget class="my_combobox" name="udpAdress">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>25</y>
      <width>171</width>
      <height>30</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>udpUse</zorder>
   <zorder>label_92</zorder>
   <zorder>label_83</zorder>
   <zorder>UDP_RS485_stacked</zorder>
   <zorder>udpAdress</zorder>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>my_combobox</class>
   <extends>QComboBox</extends>
   <header>my_combobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
